scripts/funcTests/runFuncTests.sh

```diff
@@ -75,16 +81,16 @@ then
 fi
 
 # restore packages
-echo "$DOTNET msbuild build/build.proj /t:Restore /p:VisualStudioVersion=15.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta"
-$DOTNET msbuild build/build.proj /t:Restore /p:VisualStudioVersion=15.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta
+echo "dotnet msbuild build/build.proj /t:Restore /p:VisualStudioVersion=16.0 /p:Configuration=Release /p:BuildNumber=1 /p:ReleaseLabel=beta"
```

Comment:
why run dotnet instead of $dotnet?

Reply:
$DOTNET points to cli folder, which has the latest but not stable testing SDK version.
So we want to use the stable SDK version(installed by yaml task) to build.

Reply:
The yaml task installed the stable version of SDK and added the path to PATH. So if I use 'dotnet', it will resolve the stable version.